Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\Lib\tkinter\__init__.py", line 1968, in __call__
    return self.func(*args)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\Antonio\Documents\GitHub\SDI_Tools\BulkLoads.py", line 349, in form
    formatFile(voltList)
  File "C:\Users\Antonio\Documents\GitHub\SDI_Tools\BulkLoads.py", line 166, in formatFile
    for i in range(len(max([vs,ps,ams], default=(), key=len))):
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'defaults' is an invalid keyword argument for max()
